### DISCUSSION TITLE
Supported Idenitities

### DISCUSSION THREAD

**Discussion by deepanshug** [microsoft/presidio#736] [General]
Hi, 

I just wanted to check if there is a plan for extending supported entities. I understand that we can write our own PII entities but just checking if there is something in pipeline for extending this library from your side

**Comment by omri374** [comment]
Hi @deepanshug, great question. We are adding new entities every now and then, but Presidio is mainly meant to be a toolbox allowing users to extend and customize it to their needs, including adding new types of recognizers. Every organization has different requirements when it comes to detecting PII entities.

Some references for adding new recognizers:
1. [Support the detection of new PII types](https://microsoft.github.io/presidio/analyzer/adding_recognizers/).
2. [Examples of customization in Presidio Analyzer](https://microsoft.github.io/presidio/samples/python/customizing_presidio_analyzer/).
3. [Best practices for developing recognizers](https://microsoft.github.io/presidio/analyzer/developing_recognizers/).
